app/src/main/res/xml/preferences.xml

```diff
@@ -109,6 +109,14 @@
             android:key="prefHardwareForwardButtonSkips"
             android:summary="@string/pref_hardwareForwardButtonSkips_sum"
             android:title="@string/pref_hardwareForwardButtonSkips_title"/>
+        <Preference
```

Comment:
I never said so.  My question is why you use ``Preference`` and not ``MaterialListPreference`` just like ``prefSmartMarkAsPlayedSecs``? Makes the dialog lists look inconsistent